index.js

```diff
@@ -125,7 +125,11 @@ P.prototype._onviewbox = function (bbox, zoom, cb) {
   self._getDb(function (db) {
     boxes.forEach(bbox => {
       db.query(bbox, { trace })
-        .then(q => self._loadQuery(bbox, q))
+        .then(async (q) => {
+          var now = performance.now()
+          await self._loadQuery(bbox, q)
```

Comment:
Testing this on the url `http://192.168.10.120:9966/#bbox=7.56509,47.5267,7.64573,47.56638` (zoom level ~13) takes over 60 seconds!